repos/spack_repo/builtin/packages/libxslt/package.py

```diff
@@ -74,14 +93,12 @@ def configure_args(self):
 
         return args
 
-    @run_after("install")
-    @on_package_attributes(run_tests=True)
-    def import_module_test(self):
-        if self.spec.satisfies("+python"):
-            with working_dir("spack-test", create=True):
-                python("-c", "import libxslt")
 
-    def patch(self):
-        # Remove flags not recognized by the NVIDIA compiler
-        if self.spec.satisfies("%nvhpc"):
-            filter_file("-Wmissing-format-attribute", "", "configure")
+class CMakeBuilder(CMakeBuilder):
+    def cmake_args(self):
+        return [
+            self.define_from_variant("LIBXSLT_WITH_PYTHON", "python"),
+            self.define_from_variant("LIBXSLT_WITH_CRYPTO", "crypto"),
+            self.define("DLIBXSLT_WITH_MODULES", False),
+            self.define("DLIBXSLT_WITH_TESTS", False),
```

Comment:
Should this be this?
```suggestion
            self.define("LIBXSLT_WITH_MODULES", False),
            self.define("LIBXSLT_WITH_TESTS", False),
```

Reply:
Yup, I had changed it at some point and missed those. Thanks!